### ISSUE TITLE
scientist upgrades and drones

### ISSUE THREAD

**Issue by vlad-ultramarine** [PavelZinchenko/event-horizon-frontier-builds#131]
Улучшения модулей у ученого делает эти модули лучше у дронов которых использует игрок? (и возможно у врагов?)
Вопрос возник поскольку у дронов почти нет модулей специально для них (из я знаю только двигатель) и 90% модулей в сборках дронов это обычные доступные игроку модули

**Comment by PavelZinchenko** [comment]
Нет, модули у дронов не улучшаются. Думаю это будет слишком, если улучшить отсек дронов на +200% урона и в добавок еще и оружие дрона +200%
